http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/AppServiceInterp.scala

```diff
@@ -699,32 +700,20 @@ final class LeoAppServiceInterp[F[_]: Parallel](config: AppServiceConfig,
 
       aksClusterName <- getLandingZoneResourceName(lzResourcesByPurpose,
                                                    "Microsoft.ContainerService/managedClusters",
-                                                   SHARED_RESOURCE.toString
+                                                   SHARED_RESOURCE
       )
       batchAccountName <- getLandingZoneResourceName(lzResourcesByPurpose,
                                                      "Microsoft.Batch/batchAccounts",
-                                                     SHARED_RESOURCE.toString
-      )
-      relayNamespace <- getLandingZoneResourceName(lzResourcesByPurpose,
-                                                   "Microsoft.Relay/namespaces",
-                                                   SHARED_RESOURCE.toString
+                                                     SHARED_RESOURCE
       )
+      relayNamespace <- getLandingZoneResourceName(lzResourcesByPurpose, "Microsoft.Relay/namespaces", SHARED_RESOURCE)
       storageAccountName <- getLandingZoneResourceName(lzResourcesByPurpose,
                                                        "Microsoft.Storage/storageAccounts",
-                                                       SHARED_RESOURCE.toString
-      )
-      vnetName <- getLandingZoneResourceName(lzResourcesByPurpose,
-                                             "Microsoft.Network/virtualNetworks",
-                                             SHARED_RESOURCE.toString
-      )
-      batchNodesSubnetName <- getLandingZoneResourceName(lzResourcesByPurpose,
-                                                         "Microsoft.Network/virtualNetworks/subnets",
-                                                         WORKSPACE_BATCH_SUBNET.toString
-      )
-      aksSubnetName <- getLandingZoneResourceName(lzResourcesByPurpose,
-                                                  "Microsoft.Network/virtualNetworks/subnets",
-                                                  AKS_NODE_POOL_SUBNET.toString
+                                                       SHARED_RESOURCE
       )
+      vnetName <- getLandingZoneResourceName(lzResourcesByPurpose, "DeployedSubnet", AKS_NODE_POOL_SUBNET, true)
```

Comment:
I see, good catch with the vnet being returned only as a parent resource for subnets!